addons/base_partner_address/models/res_partner.py

```diff
@@ -131,18 +132,37 @@ def create(self, vals):
                 next_by_code('res.partner') or '-'
         if not vals.get('lang', False):
             vals['lang'] = 'de_DE'
+        for field in CONTACTS:
+            field_ids = 'partner_%s_ids' % field
+            if len(vals.get(field_ids, 0)) == 1:
```

Comment:
@kaerdsar If I install the module in a fresh created DB with demo data and german language I got an error here:

```
  ...
    return old_api(self, *args, **kwargs)
  File "/var/www/openerp/addons/base/ir/ir_model.py", line 1066, in _update
    res_id = model_obj.create(cr, uid, values, context=context)
  File "/var/www/openerp/api.py", line 268, in wrapper
    return old_api(self, *args, **kwargs)
  File "/var/www/openerp/api.py", line 372, in old_api
    result = method(recs, *args, **kwargs)
  File "/var/www/addons/base_partner_address/models/res_partner.py", line 137, in create
    if len(vals.get(field_ids, 0)) == 1:
ParseError: "object of type 'int' has no len()" while parsing /var/www/addons/portal/portal_demo.xml:6, near
<record id="partner_demo_portal" model="res.partner">
    <field name="name">Demo Portal User</field>
    <field name="email">demo.portal@yourcompany.example.com</field>
    <field name="supplier" eval="False"/>
    <field name="customer" eval="True"/>
    <field name="notify_email">none</field>
    <field name="city">Vivegnis</field>
    <field name="zip">4683</field>
    <field name="country_id" ref="base.be"/>
    <field name="street">Rue Cesar de Paepe, 43</field>
</record>
```